### DISCUSSION TITLE
Longhorn volume in degraded state - v1.23.10+rke2r1[QUESTION]

### DISCUSSION THREAD

**Discussion by zohebs341** [longhorn/longhorn#4545] [Q&A]
## Environment

 - Longhorn version: rancher/mirrored-longhornio-longhorn-engine:v1.3.1
 - Kubernetes version: v1.23.10+rke2r1
 - Node config
   - OS type and version  Ubuntu 18.04 (LTS) x64
   - CPU per node: 4
   - Memory per node: 8 GB Memory
   - Disk type 160 GB Disk
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal): It's a VM from digital ocean

## Additional context

I can see, PVC got attached to pod and even I've verified by exec into Pod. I can see 8Gb volume being assigned. But in longhorn UI, it is saying degarded.

And longhorn-manager logs:

time="2022-09-08T12:19:51Z" level=debug msg="Found 0 node has at least one schedulable disk" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=k8s-worker-node-02 owner=k8s-worker-node-02 state=attached volume=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8
time="2022-09-08T12:19:51Z" level=error msg="There's no available disk for replica pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8-r-fa92b416, size 8589934592"
time="2022-09-08T12:19:51Z" level=error msg="unable to schedule replica" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=k8s-worker-node-02 owner=k8s-worker-node-02 replica=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8-r-fa92b416 state=attached volume=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8
time="2022-09-08T12:19:59Z" level=debug msg="Found 0 node has at least one schedulable disk" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=k8s-worker-node-02 owner=k8s-worker-node-02 state=attached volume=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8
time="2022-09-08T12:19:59Z" level=error msg="There's no available disk for replica pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8-r-fa92b416, size 8589934592"
time="2022-09-08T12:19:59Z" level=error msg="unable to schedule replica" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=k8s-worker-node-02 owner=k8s-worker-node-02 replica=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8-r-fa92b416 state=attached volume=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8
time="2022-09-08T12:19:59Z" level=debug msg="Found 0 node has at least one schedulable disk" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=k8s-worker-node-02 owner=k8s-worker-node-02 state=attached volume=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8
time="2022-09-08T12:19:59Z" level=error msg="There's no available disk for replica pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8-r-fa92b416, size 8589934592"
time="2022-09-08T12:19:59Z" level=error msg="unable to schedule replica" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=k8s-worker-node-02 owner=k8s-worker-node-02 replica=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8-r-fa92b416 state=attached volume=pvc-eb00391f-f40d-44dd-b753-8bebdbb6d0f8

**Comment by mantissahz** [comment]
Hi @zohebs341 
How many worker nodes (schedulbale) do you have in this cluster?
Longhorn would create 3 replicas on worker nodes in default.
Volume would be degraded if there were no enough replicas.

**Comment by zohebs341** [comment]
@mantissahz Thanks for your response.

It's a small 3-node cluster with one master & two worker nodes. As worker nodes are two, it got degraded.
From Longhorn UI, I've update volume replicas to "1" and it's health now.
Thank you.
